Source/HAL_TLC5946.c

```diff
@@ -3,10 +3,13 @@
 #include "main.h"
 
 // #define TLC_CONTINUOUS
+#ifndef TLC_DEVICES
 #define TLC_DEVICES 	3
+#endif
 
-#define TLC_GS_BYTES 24
-#define TLC_DC_BYTES 12
+#define TLC_DC_CHANNELS 16
+#define TLC_DC_BYTES    (TLC_DC_CHANNELS * 3 / 4)
+#define TLC_GS_BYTES    (TLC_DC_CHANNELS * 3 / 2)
```

Comment:
can we change to just `TLC_CHANNELS` pls?

Reply:
Done

Reply:
That was a pre-coffee commit, lacked critical thinking